filters/auth/oidc_test.go

```diff
@@ -92,26 +400,26 @@ func TestEncryptDecryptState(t *testing.T) {
 }
 
 func TestOidcValidateAllClaims(t *testing.T) {
-	oidcFilter, err := makeTestingFilter([]string{"uid", "email", "hd"})
+	oidcFilter, err := makeTestingFilter([]string{"uid", "test", "email", "test@example.org"})
```

Comment:
see my comment in oidc.go about the claims parameter, I don't think it should contain values, only keys.

Reply:
fixed